packages/components/package.json

```diff
@@ -2,44 +2,76 @@
   "name": "@kaizen/components",
   "version": "1.4.26",
   "description": "Kaizen component library",
-  "main": "dist/index.js",
-  "module": "dist/esm/index.js",
-  "types": "dist/index.d.ts",
+  "author": "Geoffrey Chong <geoff.chong@cultureamp.com>",
+  "homepage": "https://github.com/cultureamp/kaizen-design-system#readme",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/cultureamp/kaizen-discourse/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/cultureamp/kaizen-design-system.git"
+  },
   "files": [
     "future",
     "dist",
     "icons"
   ],
+  "main": "dist/cjs/index.js",
+  "module": "dist/esm/index.js",
+  "types": "dist/index.d.ts",
+  "exports": {
+    ".": {
+      "import": "./dist/esm/index.js",
+      "require": "./dist/cjs/index.js"
+    },
+    "./future": {
+      "import": "./dist/esm/future.js",
+      "require": "./dist/cjs/future.js"
+    }
+  },
   "scripts": {
-    "prepublish": "tsup && yarn tailwindcss",
-    "build": "yarn prepublish",
+    "prepublish": "yarn build:deps && yarn build:components && yarn postBuild && yarn dist:clean",
+    "build": "yarn build:clean && yarn prepublish",
+    "build:clean": "rimraf 'dist'",
+    "build:components": "rollup -c",
+    "build:deps": "yarn workspace @kaizen/design-tokens build && yarn workspace @kaizen/tailwind build",
     "tailwindcss": "tailwindcss -i ./src/styles.css -o ./dist/styles.css",
-    "build:watch": "yarn clean && yarn prepublish --watch",
-    "clean": "rimraf 'dist'"
+    "postBuild": "yarn dist:copy-tailwind && yarn dist:combine-styles && yarn dist:postcss",
+    "dist:copy-tailwind": "cp src/tailwind.css dist/tailwind.css",
```

Comment:
Rollup could handle this, but I don't want to keep the process of bundling components and the compilation of CSS separate for now. If the one tool can't do it all then it might be confusing which tool is in charge of what.

Reply:
> They still install the style.css as a global.

Is this still in WIP state? Or are we expecting users to use this already? If the latter, I think we're missing some sort of setup guide in our docs, aren't we? (Or did I not find it?)

Reply:
Yeah there's no general docs for this yet, working on a "need to know" basis at the moment eg. Jared's team using dropzone will start to use it.

I do also have this ticket which could be brought out of the tailwind context. https://cultureamp.atlassian.net/browse/KZN-1017

Reply:
Added another on in the All-in-one epic: https://cultureamp.atlassian.net/jira/software/c/projects/KZN/boards/491/roadmap?selectedIssue=KZN-1082

Reply:
Thanks!